pytorch_lightning/trainer/trainer.py

```diff
@@ -87,6 +89,7 @@ def __init__(
             num_sanity_val_steps=5,
             truncated_bptt_steps=None,
             resume_from_checkpoint=None,
+            profiler=None
```

Comment:
i thought the options were:
None = don't use
True = Use a default profiler
Object = A profiler with your own options for it

Reply:
ah that's right, i'll update that tonight